packages/block/src/block.ts

```diff
@@ -217,10 +220,22 @@ export class Block {
 
     // First try to load header so that we can use its common (in case of setHardfork being activated)
     // to correctly make checks on the hardforks
-    const [headerData, txsData, uhsData, withdrawalBytes, requestBytes, executionWitnessBytes] =
```

Comment:
we can't do a simple assignment because this sequence is dependent on the eip activation